src/06-blackjack/Blackjack/Blackjack.Client.Console/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Blackjack.Client.Console
+{
+    using Blackjack.Core;
+    using System;
+    using System.Collections.Generic;
+
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.Title = "♠♥♣♦ Blackjack";
+            Console.WriteLine("Welcome to the game of Blackjack");
+            Console.WriteLine();
+
+            var player = "player";
+            var dealer = "dealer";
+
+            var deck = new Deck();
+
+            Console.WriteLine($"You have been dealt: {deck.TakeCard(player).Description} {deck.TakeCard(player).Description}");
+            Console.WriteLine();
+            while (true)
+            {
+                Console.WriteLine("What do you want to do?");
+                Console.WriteLine("----Choose 1 - Take antoher card");
+                Console.WriteLine("----Choose 2 - Finish");
+                Console.WriteLine();
+
+                var answer = int.Parse(Console.ReadLine());
+                if (answer==1)
+                {
+                    Console.WriteLine($"You have been dealt: {deck.TakeCard(player).Description}");
+                    Console.WriteLine($"Your hand: {deck.ShowCards()}");
```

Comment:
`deck.ShowCards` mean that I have a deck of cards and I show all of them... Not player cards, but all cards which are in the deck..